portal-frontend/src/app/features/notice-of-intents/edit-submission/additional-information/additional-information.component.html

```diff
@@ -67,7 +67,19 @@ <h2>Additional Proposal Information</h2>
     <form [formGroup]="structuresForm">
       <mat-label>Provide the total floor area (m<sup>2</sup>) for each of the proposed structure(s)</mat-label>
       <div class="scrollable">
-        <table mat-table [dataSource]="structuresSource">
+        <div *ngIf="isMobile">
+          <app-structure-mobile-card            
+            *ngFor="let structure of proposedStructures; let last = last; index as i"
+            [structure]="structure"
+            [isLast]="last"
+            [isReviewStep]="true"
```

Comment:
Is "isReviewStep" same as other mobile components we have? In other mobile components like "parcel-owner-mobile-card" or "naru-residence-mobile-card", the 'isReviewStep' input is to distinguish between step 8 and other steps in submission to remove the kebab menu (3 dot menu) and change the background to be transparent.

Also, in edit submission, we assign false value to isReviewStep since it is not in the review step (step 8).

Reply:
My Initial plan was to re-use it. But, since the design changes on review part, I'll remove this property.

Reply:
How much does the design change on review part?